nss-day-cohort-60/rare-client-tenacious-tators
# Jl unsubscribe client

Closes Ticket #3 


# Dynamically renders subscribe/unsubscribe button and deletes subscriptions

---
# NOTES
Your current API may have multiple rows of the same subscription with different primary keys. This functionality is based on fetching a singular row. You may need to unsubscribe multiple times if there are duplicate subscriptions in the API.

# Steps to Test

1. Fetch the jl-unsubscribe-client branch and switch to it
2. Navigate to http://localhost:3000/users
3. Navigate to user profile
4. Click Unsubscribe/Subscribe
5. Navigate back to the same user profile
6. Click Unsubscribe/Subscribe (should be a new button - see NOTES above)
7. Repeat steps 5-6
